app/components/projects/table_component.rb

```diff
@@ -153,6 +153,10 @@ def projects_with_level(projects, &)
       end
     end
 
+    def favored_projects
```

Comment:
Naming suggestion: `favored_project_ids` as we do not have a collection of the project instances here.

```suggestion
    def favored_project_ids
```